jupyterhub/auth.py

```diff
@@ -635,6 +635,24 @@ def delete_user(self, user):
         """
         self.allowed_users.discard(user.name)
 
+    manage_groups = Bool(
+        config=True, help="Let authenticator manage user groups"
+    )
+
+    @default('manage_groups')
+    def _manage_groups_default(self):
+        return False
+
```

Comment:
Don't need a dynamic default method for a static value:

```suggestion
    manage_groups = Bool(
        False,
        config=True,
        help="""Let authenticator manage user groups
        
        Authenticator must implement get_user_groups for this to be useful.
        """
    )
```